applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -62,4 +67,32 @@
         </div>
         @fragments.contentFooter(interactive, page.related, isPaidContent = isPaidContent)
     }
+
+    <script>
+        require(['Promise'], function (Promise) {
```

Comment:
I had hoped that for users with native Promise this becomes a synchronous require call but then I don't really know if that's even allowed in the AMD spec :stuck_out_tongue: